kaggle_environments/main.py

```diff
@@ -23,7 +23,7 @@
 parser = argparse.ArgumentParser(description="Kaggle Simulations")
 parser.add_argument(
     "action",
-    choices=["list", "evaluate", "run", "step", "load", "act", "http-server"],
+    choices=["list", "evaluate", "run", "step", "load", "act", "dispose", "http-server"],
```

Comment:
The new dispose option is only used to write the closing json token to the log file when an episode has completed in http-server mode.

Reply:
I don't really like this pattern long-term but it should work fine for now.